index.js

```diff
@@ -22,16 +16,25 @@ async function run() {
 		const {stdout: t} = await execFile('git', ['tag', '-l', '--sort=-creatordate']);
 		const tags = t.split('\n').filter(Boolean).map(tag => tag.trim());
 
+		if (tags.length === 0) {
+			core.setOutput('There is nothing to be done here. Exiting!');
```

Comment:
```suggestion
			core.info('There is nothing to be done here. Exiting!');
			return;
```

The output should not be a message, but just data, if any: https://github.com/actions/toolkit/tree/master/packages/core#inputsoutputs